src/java.desktop/share/classes/java/awt/GridBagLayout.java

```diff
@@ -277,70 +277,64 @@
  * Here is the code that implements the example shown above:
  *
  * <hr><blockquote><pre>
- * import java.awt.*;
- * import java.util.*;
- * import java.applet.Applet;
+ * import java.awt.Button;
+ * import java.awt.EventQueue;
+ * import java.awt.Frame;
+ * import java.awt.GridBagConstraints;
+ * import java.awt.GridBagLayout;
  *
- * public class GridBagEx1 extends Applet {
+ * public class GridBagLayoutExample {
  *
- *     protected void makebutton(String name,
- *                               GridBagLayout gridbag,
- *                               GridBagConstraints c) {
- *         Button button = new Button(name);
- *         gridbag.setConstraints(button, c);
- *         add(button);
- *     }
+ *  static void addButton(String name,
+ *                        GridBagLayout gridbag,
+ *                        GridBagConstraints c,
+ *                        Frame frame) {
+ *     Button button = new Button(name);
+ *     gridbag.setConstraints(button, c);
+ *     frame.add(button);
+ *  }
  *
- *     public void init() {
- *         GridBagLayout gridbag = new GridBagLayout();
- *         GridBagConstraints c = new GridBagConstraints();
+ *  public static void main(String[] args) throws Exception {
  *
- *         setFont(new Font("SansSerif", Font.PLAIN, 14));
- *         setLayout(gridbag);
+ *    EventQueue.invokeAndWait(() -> {
+ *       Frame frame = new Frame("GridBagLayout");
+ *       GridBagLayout gridbag = new GridBagLayout();
+ *       GridBagConstraints c = new GridBagConstraints();
+ *       frame.setLayout(gridbag);
```

Comment:
The `main` method should be indented by 4 spaces to the right, it's inside `GridBagLayoutExample` class.

Reply:
will adjust